python/orca/src/bigdl/orca/data/image/voc_dataset.py

```diff
@@ -72,24 +76,25 @@ def _load_items(self, splits_names):
                 continue
         return img_ids
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self._imgid_items)
 
-    def __iter__(self):
+    def __iter__(self) -> zip:
         img_path = [self._image_path.format(*img_id) for img_id in self._imgid_items]
         return zip(img_path, self._im_anno)
 
-    def __getitem__(self, idx):
+    def __getitem__(self, idx: int) -> Tuple["ndarray", "ndarray"]:
         img_id = self._imgid_items[idx]
         img_path = self._image_path.format(*img_id)
         if img_path in self._im_cache:
-            img = self._im_cache
+            img = self._im_cache[img_path]
         else:
-            img = self._read_image(img_path)
+            img = self._read_image(img_path)  # type: ignore
 
         return img, self._im_anno[idx]
 
-    def _load_label(self, idx):
+    @no_type_check
```

Comment:
what does this mean? is it different from type: ignore?

Reply:
> what does this mean? is it different from type: ignore?
This annotation just ignores type checking of this method, while type:ignore only works for one line.